api/src/pcapi/core/finance/api.py

```diff
@@ -877,12 +879,17 @@ def generate_invoice_file(invoice_date: datetime.date = datetime.date.today()) -
         .join(models.Invoice.cashflows)
         .join(models.Cashflow.pricings)
         .join(models.Pricing.lines)
+        .join(models.Invoice.businessUnit)
+        .join(
+            BusinessUnitVenue,
+            models.BusinessUnit.siret == BusinessUnitVenue.siret,
+        )
```

Comment:
Je pense qu'on devrait pouvoir faire sans l'alias, comme ça :

```suggestion
        .join(
            models.BusinessUnit.venue,
            models.BusinessUnit.siret == offerers_models.Venue.siret,
        )
```

On doit utiliser un alias dans la fonction que je t'ai indiquée, car on y fait 2 jointures sur la table `venue`, et l'alias permet de les distinguer. Ici, on ne fait qu'une jointure sur cette table, donc l'alias ne devrait pas être nécessaire.